packages/hop-node/src/db/GasCostDb.ts

```diff
@@ -43,12 +44,30 @@ class GasCostDb extends BaseDb {
     }
   }
 
+  async migration () {
+    this.logger.debug('GasCostDb migration started')
+    const entries = await this.getKeyValues()
+    this.logger.debug(`GasCostDb migration: ${entries.length} entries`)
+    for (const entry of entries) {
+      let transactionType: GasCostTransactionType
```

Comment:
I think there should be another check here so it doesn't re-set values on re-runs

```ts
if (typeof entry.value.transactionType !== 'undefined') {
 continue
}
```